mlflow/pyfunc/scoring_server.py

```diff
@@ -181,24 +179,9 @@ def transformation():  # pylint: disable=unused-variable
     return app
 
 
-def _safe_local_path(local_path):
-    # NB: Since mlflow 1.0, mlflow.pyfunc.load_model expects uri instead of local path. Local paths
-    # work, however absolute windows path don't because the drive is parsed as scheme. Since we do
-    # not control the version of mlflow (the scoring server version matches the version of mlflow
-    # invoking the scoring command, the rest of mlflow comes from the model environment) we check
-    # the mlflow version at run time and convert local path to file uri to ensure platform
-    # independence.
-    from mlflow.version import VERSION
-    is_recent_version = VERSION.endswith("dev0") or int(VERSION.split(".")[0]) >= 1
```

Comment:
What happened to this logic? Were you able to test this on Windows?

Reply:
The logic has been moved to the caller. The caller is supposed to convert local path to file uri.  Actually, good point, I have marked the cli test large and large tests do not run on windows. Hmm, I'll probably put it back to small then even though it's a slower one.

Reply:
Oh and also - we're not trying to keep backwards compatibility to pre 1.0 release anymore so we can simply transform path to  file URI and be sure it works on all platforms.